src/rtc/defaults/rtc_s1.yaml

```diff
@@ -110,6 +110,14 @@ runconfig:
             # Apply thermal noise correction
             apply_thermal_noise_correction: True
 
+            # Compute geolocation correction LUTs and apply it
+            apply_correction_luts: True
+
+            # slant range spacing of the correction LUT in meters
+            correction_lut_range_spacing: 200
+            # Azimuth time spacing of the correction LUT in seconds
+            correction_lut_azimuth_spacing: 0.028
```

Comment:
I'd prefer to have these fields as number of az./rg. looks instead of rg spacing in meters and az. spacing in seconds. @hfattahi , do you see any problem with that? It's difficult to know how many looks 200m or 0.028 seconds represent.

Reply:
Another note: In case that we decide to call `topo` once as you suggested in the comment above, I think the spacing option in runconfig would not be relevant because layover shadow mask has to be run on full radargrid resolution?